src/eh1/digital.rs

```diff
@@ -132,6 +150,10 @@ pub enum TransactionKind {
     Set(State),
     /// Get the pin state
     Get(State),
+    /// Toggle for an expected toggle of the pin
```

Comment:
That sentence looks a bit strange... Maybe a relict from some rewording?

I'll change to `Toggle the pin state`.